src/pkg/cli/debug_test.go

```diff
@@ -64,17 +65,42 @@ func TestQueryHasProject(t *testing.T) {
 	provider := MustHaveProjectNameQueryProvider{}
 	fabricClient := MockDebugFabricClient{}
 
-	if err := Debug(context.Background(), fabricClient, provider, "etag", project, []string{"service"}); err != nil {
+	if err := Debug(context.Background(), fabricClient, provider, "etag", project, []string{"service"}, nil); err != nil {
 		t.Errorf("expected no error, got %v", err)
 	}
 
 	project.Name = ""
 
-	if err := Debug(context.Background(), fabricClient, provider, "etag", project, []string{"service"}); err == nil {
+	if err := Debug(context.Background(), fabricClient, provider, "etag", project, []string{"service"}, nil); err == nil {
 		t.Error("expected error, got nil")
 	} else {
 		if err.Error() != "project name is missing" {
 			t.Errorf("expected error %q, got %q", "project name is missing", err.Error())
 		}
 	}
 }
+
+func TestDebugProject(t *testing.T) {
+	project := &compose.Project{
+		Name:         "project",
+		WorkingDir:   "workingdir",
+		Environment:  types.Mapping{},
+		ComposeFiles: []string{"composefile"},
+	}
+
+	loadErr := errors.New("load error")
+	provider := MustHaveProjectNameQueryProvider{}
+	fabricClient := MockDebugFabricClient{}
+
+	if err := Debug(context.Background(), fabricClient, provider, "etag", project, []string{"service"}, loadErr); err != nil {
+		t.Errorf("expected no error, got %v", err)
+	}
+
+	if err := Debug(context.Background(), fabricClient, provider, "", project, []string{"service"}, nil); err != nil {
+		if err.Error() == "no information to use for debugger" {
+			return
+		}
+	}
+
+	t.Error("expected error, got none")
```

Comment:
little odd to have this outside of the `if`, will make it cumbersome when we add tests later:
```suggestion
	t.Run("without load error", func(t *testing.T) {
		err := Debug(context.Background(), fabricClient, provider, "", project, []string{"service"}, nil)
		if err == nil {
			t.Fatal("expected error, got none")
		}
		if err.Error() != "no information to use for debugger" {
			t.Error("expected: no information to use for debugger")
		}
	})
```